packages/playwright/types/test.d.ts

```diff
@@ -18,6 +18,8 @@
 import type { APIRequestContext, Browser, BrowserContext, BrowserContextOptions, Page, LaunchOptions, ViewportSize, Geolocation, HTTPCredentials, Locator, APIResponse, PageScreenshotOptions } from 'playwright-core';
 export * from 'playwright-core';
 
+import type { TestCase } from './testReporter';
```

Comment:
I'm not sure whether it is okay to simply cross reference `TestCase` here…

Reply:
It is and it isn't, but let's solve other bits before resolving it.